polyply/src/backmap.py

```diff
@@ -60,72 +60,77 @@ def orient_template(meta_molecule, current_node, template, built_nodes):
     # 1. find neighbours at meta_mol level
     neighbours = nx.all_neighbors(meta_molecule, current_node)
     current_resid = meta_molecule.nodes[current_node]["resid"]
-
-    # 2. find connecting atoms at low-res level
-    edges = []
-    ref_nodes = []
-    for node in neighbours:
-        resid = meta_molecule.nodes[node]["resid"]
-        edge = find_connecting_edges(meta_molecule,
-                                     meta_molecule.molecule,
-                                     (node, current_node))
-        edges += edge
-        ref_nodes.extend([node]*len(edge))
-
-    # 3. build coordinate system
-    ref_coords = np.zeros((3, len(edges)))
-    opt_coords = np.zeros((3, len(edges)))
-
-    for ndx, edge in enumerate(edges):
-        for atom in edge:
-            resid = meta_molecule.molecule.nodes[atom]["resid"]
-            if resid == current_resid:
-                current_atom = atom
+    if len(list(neighbours)) > 0:
```

Comment:
I don't see the point of this change. `If len(neighbours) == 0`, then `for node in neighbours` is a noop, `edges` will still be empty, and `for ndx, edge in enumerate(edges)` will also be a noop.